backend/spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -59,6 +59,33 @@
         expect(assigns(:collection)).to eq [user]
       end
     end
+
+    context "when Spree.user_class have a different namespace than Spree" do
+      class UserModel < ApplicationRecord
+        self.table_name = 'spree_users'
+        include Spree::UserMethods
+      end
+
+      around do |example|
+        actual_user_class = Spree.user_class
```

Comment:
Oh, my fault. This should fix the CI, right?
```suggestion
        actual_user_class = Spree.user_class.name
```